<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>主窗口</string>
  </property>
  <widget class="Clock" name="widgetClock" native="true">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>88</y>
     <width>162</width>
     <height>162</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="Calendar" name="widgetCalendar" native="true">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>60</y>
     <width>210</width>
     <height>216</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="DigitalClock" name="lcdNumberDigClock">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>3</number>
   </property>
   <property name="numDigits">
    <number>11</number>
   </property>
   <property name="value" stdset="0">
    <double>1111111111.000000000000000</double>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewNotifier">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>64</y>
     <width>264</width>
     <height>210</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDummy">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Dummy</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonVoip">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>81</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


户户对讲</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonInfoMan">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


信息管理</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonSecurityMan">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


安防管理</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonVideoMon">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>81</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


视频监控</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonIntercomMan">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>81</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


对讲管理</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonCallCenter">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


呼叫中心</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonTerminalHelp">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


终端求助</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonCallElevator">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>81</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


呼叫电梯</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonInterFreeSetting">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>320</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>81</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


免扰设置</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonPhoto">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>81</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


数码相框</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonSystemSetting">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>81</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


系统设置</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonSendImg">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>81</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


发送图片</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonHomeControl">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>81</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


家居控制</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButtonOther">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>330</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>81</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>


其他</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Clock</class>
   <extends>QWidget</extends>
   <header>clock.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DigitalClock</class>
   <extends>QLCDNumber</extends>
   <header>digitalclock.h</header>
  </customwidget>
  <customwidget>
   <class>Calendar</class>
   <extends>QWidget</extends>
   <header>calendar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonDummy</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
